connectors/README.md

```diff
@@ -23,64 +27,58 @@ on a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster.
+    and `spark-bigquery-connector-version` metadata values:
 
-    ```
+    ```shell
     REGION=<region>
     CLUSTER_NAME=<cluster_name>
     gcloud dataproc clusters create ${CLUSTER_NAME} \
         --region ${REGION} \
         --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/connectors/connectors.sh \
-        --metadata gcs-connector-version=2.2.0 \
         --metadata bigquery-connector-version=1.2.0 \
-        --metadata spark-bigquery-connector-version=0.20.0
+        --metadata spark-bigquery-connector-version=0.21.0
     ```
 
--   to update connector by specifying URL, use `gcs-connector-url`,
-    `bigquery-connector-url`and `spark-bigquery-connector-url` metadata values:
+*   to update connector by specifying URL, use `bigquery-connector-url`and
+    `spark-bigquery-connector-url` metadata values:
 
-    ```
+    ```shell
     REGION=<region>
     CLUSTER_NAME=<cluster_name>
     gcloud dataproc clusters create ${CLUSTER_NAME} \
         --region ${REGION} \
         --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/connectors/connectors.sh \
-        --metadata gcs-connector-url=gs://path/to/custom/gcs/connector.jar \
         --metadata bigquery-connector-url=gs://path/to/custom/hadoop/bigquery/connector.jar \
         --metadata spark-bigquery-connector-url=gs://path/to/custom/spark/bigquery/connector.jar
     ```
 
-This script downloads specified Google Cloud Storage connector, Hadoop BigQuery
-connector and Spark BigQuery connector and deletes an old version of these
-connectors if they were installed.
+This script downloads specified Hadoop BigQuery connector and Spark BigQuery
+connector and deletes an old version of these connectors if they were installed.
 
 To specify connector version, find the connector version on the
 [Hadoop connectors releases page](https://github.com/GoogleCloudDataproc/hadoop-connectors/releases)
 and
 [Spark BigQuery connector releases page](https://github.com/GoogleCloudDataproc/spark-bigquery-connector/releases),
-and set it as the `gcs-connector-version`, `bigquery-connector-version` or
+and set it as the `bigquery-connector-version` or
 `spark-bigquery-connector-version` metadata key value.
 
-If only one connector version is specified (Google Cloud Storage, Hadoop
-BigQuery or Spark BigQuery) then only this connector will be updated.
+If only one connector version is specified (Hadoop BigQuery or Spark BigQuery)
+then only this connector will be updated.
 
 For example:
 
-*   if Google Cloud Storage connector 2.2.0 version is specified and neither
-    Hadoop BigQuery connector not Spark BigQuery connector versions are
-    specified, then only Google Cloud Storage connector will be updated to 2.2.0
-    version:
+*   if Spark BigQuery connector 0.21.1 version is specified and Hadoop BigQuery
+    connector version is specified, then only Spark BigQuery connector version
```

Comment:
Change to `Hadoop BigQuery connector version is not specified`

Reply:
Done.